### PR TITLE
Convert Circuit into a Parachain - Closes #158

### LINKED ISSUE #158: Convert Circuit into a Parachain of "t3stnet" using Cumulus
### Description

Use substrate parachain template, update Circuit to a standalone parachain, and launch on a local "t3stnet"

Mind the onfinality docs - https://support.onfinality.io/hc/en-us/articles/4407873248025-Create-a-custom-relay-chain-and-parachain-in-OnFinality

### Acceptance Criteria

- [x] Circuit runtime has all the necessary cumulus and xcm pallets installed
- [x] Circuit runtime has every circuit pallet installed
- [x] Circuit runtime and node compile correctly
- [ ] Circuit is compatible with Polkadot v0.9.8 relay chain

### PR DESCRIPTION
**Pull request by beqaabu**
Closes #158

### CODE REVIEW

File: circuit-parachain/node/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+[package]
+name = "parachain-template-node"
```
Comment (vedhavyas) [inline circuit-parachain/node/Cargo.toml:2]: lets rename this

Comment (beqaabu) [reply circuit-parachain/node/Cargo.toml]: updated on the following commit

File: circuit-parachain/.dockerignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.git
+**/target/
+**/*.txt
+**/*.md
+/docker/
+!/target/release/polkadot-collator
+
+# dotfiles in the repo root
+/.*
```
Comment (MaciejBaj) [inline circuit-parachain/.dockerignore:9]: missing EOF line - correct in other files too

File: circuit-parachain/node/src/command.rs [hunk 3]
```diff
@@ -0,0 +1,405 @@
+use crate::{
+	chain_spec,
+	cli::{Cli, RelayChainCli, Subcommand},
+	service::{new_partial, TemplateRuntimeExecutor},
```
Comment (MaciejBaj) [inline circuit-parachain/node/src/command.rs:4]: Rename `TemplateRuntimeExecutor` -> `CircuitRuntimeExecutor`

File: circuit-parachain/node/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,94 @@
+[package]
+name = "t3rn-parachain-node"
+version = "0.1.0"
+authors = ["Anonymous"]
+description = "A new Cumulus FRAME-based Substrate Node, ready for hacking together a parachain."
+license = "Unlicense"
+homepage = "https://substrate.io"
+repository = 'https://github.com/substrate-developer-hub/substrate-parachain-template'
```
Comment (MaciejBaj) [inline circuit-parachain/node/Cargo.toml:8]: change desc - apply consistently

File: circuit-parachain/node/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,94 @@
+[package]
+name = "t3rn-parachain-node"
+version = "0.1.0"
+authors = ["Anonymous"]
+description = "A new Cumulus FRAME-based Substrate Node, ready for hacking together a parachain."
+license = "Unlicense"
```
Comment (MaciejBaj) [inline circuit-parachain/node/Cargo.toml:6]: Apache 2.0 - apply consistently

File: circuit-parachain/node/src/command.rs [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+	}
+
+	fn native_runtime_version(_: &Box<dyn ChainSpec>) -> &'static RuntimeVersion {
+		&t3rn_parachain_runtime::VERSION
+	}
+}
+
+impl SubstrateCli for RelayChainCli {
+	fn impl_name() -> String {
+		"Parachain Collator Template".into()
```
Comment (MaciejBaj) [inline circuit-parachain/node/src/command.rs:75]: Change from Template to Circuit in command.rs consistently

File: circuit-parachain/node/src/chain_spec.rs [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+			..Default::default()
+		},
+		session: t3rn_parachain_runtime::SessionConfig {
+			keys: invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                 // account id
+						acc,                         // validator id
+						template_session_keys(aura), // session keys
```
Comment (MaciejBaj) [inline circuit-parachain/node/src/chain_spec.rs:205]: change template naming in chain_spec

File: circuit-parachain/node/src/command.rs [hunk 8]
```diff
@@ -0,0 +1,405 @@
...
+use sp_runtime::traits::Block as BlockT;
+use std::{io::Write, net::SocketAddr};
+
+fn load_spec(
+	id: &str,
+	para_id: ParaId,
+) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+	Ok(match id {
+		"dev" => Box::new(chain_spec::development_config(para_id)),
+		"template-rococo" => Box::new(chain_spec::local_testnet_config(para_id)),
```
Comment (MaciejBaj) [inline circuit-parachain/node/src/command.rs:27]: @vedhavyas fetch spec for testnet from remote url?

**Review by MaciejBaj — CHANGES_REQUESTED** [review]
also, remove `pallet-template` from Cargo + Cargo.lock

**Pull request closed without merging**